README.md

```diff
@@ -1,7 +1,40 @@
-# hypertrace-trace-enricher
+# Hypertrace Trace Enricher
 ###### org.hypertrace.traceenricher
 
 [![CircleCI](https://circleci.com/gh/hypertrace/hypertrace-trace-enricher.svg?style=svg)](https://circleci.com/gh/hypertrace/hypertrace-trace-enricher)
 
 A streaming job that enriches the incoming spans with identified entities like API, Service, Backend, etc.
```

Comment:
I think we are calling it Endpoint instead of API. Please ignore if thats not the case
```suggestion
A streaming job that enriches the incoming spans with identified entities like Endpoint, Service, Backend, etc.
```